README.md

```diff
@@ -1 +1,20 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## Hello everyone, I'm **Dora Gonzalez** and it is nice to virtually meet you all! ##
+
+<img src= "https://github.com/Dora-MG/hw01/blob/master/space%20buns%20.jpg?raw=true" width="350"> 
```

Comment:
I would suggest using the image syntax here instead. Other than that, it's great.

Reply:
Great note here @bamartinez ! Preferably you would have used Markdown syntax to embed your image. Since your image is already in your GitHub repo, you can simply type `![](space buns .jpg)`